test/e2e/networkpolicy_test.go

```diff
@@ -806,6 +806,162 @@ func TestIngressPolicyWithoutPortNumber(t *testing.T) {
 	}
 }
 
+func TestIngressPolicyWithEndPort(t *testing.T) {
```

Comment:
How the test works if the testbed doesn't support 1.21 yet?

Reply:
Same question, maybe we can add some mechanism to check if the feature gate is enabled? Didn't find a clean way but a simple look into the api server process seems to do the trick https://stackoverflow.com/questions/50441452/check-if-the-feature-gate-is-enabled-disabled-in-kubernetes
```
ps aux | grep apiserver | grep feature-gates
root       15458  7.8 19.7 1449248 402504 ?      Ssl  May23 185:47 kube-apiserver --advertise-address=192.168.77.100  .............. --etcd-servers=https://127.0.0.1:2379 --feature-gates=NetworkPolicyEndPort=true...........
```

Reply:
I thought that because the `Vagrant` and `Kind` testbed have supported 1.21, we could check the test status in these two environments and ignore the Jenkins one for now. 
Do you have any ideas in mind about this issue?

Reply:
Good point. Addressed.